valve-firmware/valve-firmware.spec

```diff
@@ -0,0 +1,47 @@
+Name: valve-firmware
+# There are two source packages. Set the RPM version to mirror whichever source is newer.
+Version: 20231113.1
+Release: 1%{?dist}
+Summary: Linux firmware files for the Steam Deck OLED
+License: GPL+ and GPLv2+ and MIT and Redistributable, no modification permitted
```

Comment:
This license mirrors what `linux-firmware` uses.

https://src.fedoraproject.org/rpms/linux-firmware/blob/rawhide/f/linux-firmware.spec#_10

In SteamOS, the `linux-firmware` and `linux-firmware-neptune` packages have the same license that allows all files to be redistritubale.